instrumentation/spring/spring-rmi/javaagent/build.gradle.kts

```
@@ -0,0 +1,23 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.springframework")
+    module.set("spring-context")
+    versions.set("[5.0.4.RELEASE,5.4)")
+  }
+}
+
+dependencies {
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+
+  bootstrap(project(":instrumentation:rmi:bootstrap"))
+  testInstrumentation(project(":instrumentation:rmi:javaagent"))
+
+  library("org.springframework:spring-context:5.2.9.RELEASE")
```

by default we run the tests against the "base" version, and [there's a separate task](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/running-tests.md#executing-tests-against-the-latest-versions-of-libraries-under-instrumentation) that runs the tests against the "latest" version that it can find in maven central
```suggestion
  library("org.springframework:spring-context:5.0.4.RELEASE")
```